manifests/kubernetes/kubevirt-tekton-tasks-kubernetes.yaml

```diff
@@ -795,60 +617,10 @@ spec:
           value: $(params.deleteObjectKind)
         - name: DELETE_OBJECT_NAME
           value: $(params.deleteObjectName)
-
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: modify-data-object-task
-rules:
-  - verbs:
-      - get
-      - create
-      - delete
-    apiGroups:
-      - cdi.kubevirt.io
-    resources:
-      - datavolumes
-      - datasources
-  - verbs:
-      - create
-    apiGroups:
-      - ""
-    resources:
-      - pods
-  - verbs:
-      - get
-      - delete
-    apiGroups:
-      - ""
-    resources:
-      - persistentvolumeclaims
-
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: modify-data-object-task
-
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: modify-data-object-task
-roleRef:
-  kind: ClusterRole
-  name: modify-data-object-task
-  apiGroup: rbac.authorization.k8s.io
-subjects:
-  - kind: ServiceAccount
-    name: modify-data-object-task
 ---
 apiVersion: tekton.dev/v1
 kind: Task
 metadata:
-  annotations:
```

Comment:
Btw. where was this account defined before? I do not see it in this file.

Reply:
it is a bug. The SA is defined here, but in reality it never existed :D